### DISCUSSION TITLE
machine learning

### DISCUSSION THREAD

**Discussion by shaik23032001** [mrdbourke/pytorch-deep-learning#968] [General]
i am new to machine learning as my comany said me to learn machine learning and make a project on it after 25 days how can i learn it very fastly using pytorch i know i am in big trouble any help or guidence is appreciated

**Comment by sangeetsatheesh** [comment]
Are you familiar with python? Since  time is limited, you can focus on any one task. Linear regression would be a good start. Read up on linear regression and follow mrdbourke's pytorch lessons. I'm sure you'll be able to catch up :)
All the best!
